ifrs17/Import/1ImportScope-Identities.ipynb

```diff
@@ -299,7 +300,9 @@
                 "public interface ParentAocStep : IScope<(ImportIdentity Id, string AmountType), ImportStorage>",
                 "\n{",
                 "\n    static ApplicabilityBuilder ScopeApplicabilityBuilder(ApplicabilityBuilder builder) =>",
-                "\n                builder.ForScope<ParentAocStep>(s => s.WithApplicability<ParentAocStepForCreditRisk>(x => x.Identity.AmountType != AmountTypes.CDR));",
+                "\n                builder.ForScope<ParentAocStep>(s => // s.WithApplicability<ParentAocStepForCreditRisk>(x => x.Identity.AmountType != AmountTypes.CDR));",
+                "\n                                                    s.WithApplicability<ParentAocStepForCreditRisk>(x => !x.GetStorage().GetCdr().Contains(x.Identity.AmountType)));",
```

Comment:
Adjusting the applicability of ParentAocStep here such that it is triggered when Identity.AmountType is not CDRI **or** CDR breaks the AocStructureTest

<img width="840" alt="MicrosoftTeams-image" src="https://github.com/Systemorph/IFRS17CalculationEngine/assets/114566025/83846539-1a61-49c7-8f59-09b315fffa71">

@dtrzesniak does it make sense to adapt the applicability here in this way?